src/pages/dashboard/Explore.tsx

```diff
@@ -40,51 +55,83 @@ const StyledList = styled(List)`
 
 const Explore: FC = () => {
   const location = useLocation();
+  const [query, setQuery] = useState<null | QueryObject>(null);
   const [searchResult, setSearchResult] = useState<SearchResult>();
   const [resources, setResources] = useState<Resource[]>([]);
   const { t } = useTranslation();
   const [searchError, setSearchError] = useState(false);
+  const [page, setPage] = useState(1);
+  const history = useHistory();
 
-  const triggerSearch = async (query: string) => {
-    try {
-      const response = await searchResources(query);
-      if (response.data) {
-        setSearchError(false);
-        setSearchResult(response.data);
-        setResources(response.data.resourcesAsJson.map((resourceAsString: string) => JSON.parse(resourceAsString)));
-      } else {
-        setSearchError(true);
-      }
-    } catch (error) {
-      setSearchError(true);
-    }
+  const handlePaginationChange = (event: React.ChangeEvent<unknown>, value: number) => {
+    setPage(value);
+    const searchParams = new URLSearchParams(location.search);
+    const offset = NumberOfResultsPrPage * (value - 1);
+    offset === 0 ? searchParams.delete('offset') : searchParams.set('offset', '' + offset);
+    const url = searchParams.toString();
```

Comment:
```suggestion
    const search = searchParams.toString();
```
dette er vel bare search/query, ikke url?